Documentation/Sprint 3/reflection.tex

```diff
@@ -0,0 +1,158 @@
+\documentclass[11pt,a4paper,landscape]{article}
+\usepackage{titling,graphicx,palatino,amsmath,enumerate,booktabs,listings,color}
+\usepackage[none]{hyphenat}
+\usepackage[parfill]{parskip}
+\usepackage{pgfplots}
+\usepackage{longtable}
+\usepackage[top=2 cm, bottom= 2 cm, left=1 cm, right=1 cm]{geometry}
+\usepackage{enumitem}
+\begin{document}
+
+
+\section*{Sprint reflection Plan 3}
+Context : Multimedia services\\
+Group : Multimedia services 1\\
+
+
+
+\begin{table}[h]
+\begin{tabular}{|p{3cm}|p{5.0cm}|p{3.0cm}|p{2.2cm}|p{2.2cm}|p{2.0cm}|p{5cm}}
+\textbf{User Story} & \textbf{Task} & \textbf{Task assigned to} & \textbf{Estimated Effort per Task (Hours)} & \textbf{Actual Effort per Task (Hours)} & \textbf{Done} & \textbf{Notes}\\
+As a team member, I want to improve our documentation based on the feedback of our stakeholders.
+
+&
+\begin{enumerate}
+\item Process feedback Product Planning.
+\item Add proper Glossary for Product Planning.
+\item Review changes to Product Planning.
+\item Review changes to Emergent Architecture Design.
+\end{enumerate}
+
+& 
+\begin{enumerate}
+\item Eva Anker
+\item Gijs Weterings
+\item Jaap Heijligers
+\item Jaap Heijligers
+\end{enumerate}
+
+& 
+\begin{enumerate}
+\item 6 hours
+\item 2 hours
+\item 2 hours
+\item 2 hours
+\end{enumerate}
+
+
+&
+\begin{enumerate}
+\item 6 hours
+\item 2 hours
+\item 2 hours
+\item 2 hours
+\end{enumerate}
+&
+\begin{enumerate}
+\item Yes
+\item Yes
+\item Yes
+\item Yes
+\end{enumerate}
+&
+\begin{enumerate}
+\item 
+\item 
+\item 
+\item JW was in charge of the changes. This has been an oversight in the product planning.
+\end{enumerate}
+\\
+\end{tabular}
+\end{table}
+\begin{table}[h]
+\begin{tabular}{|p{3cm}|p{5.0cm}|p{3.0cm}|p{2.2cm}|p{2.2cm}|p{2.0cm}|p{5cm}}
+As a Scrum Master, I want my team to have a clear vision of the architecture of the system we will be building this sprint.
+&
+\begin{enumerate}
+\item Brainstorm about best architecture
+\item Prototype architectures
+\end{enumerate}
+
+&
+\begin{enumerate}
+\item Gijs Weterings
+\item Tim van der Lippe
+\end{enumerate}
+
+&
+
+\begin{enumerate}
+\item 4 hours
+\item 4 hours
+\end{enumerate}
+&
+\begin{enumerate}
+\item 26 hours
+\item 0 hours
+\end{enumerate}
+&
+\begin{enumerate}
+\item Yes
+\item No
+
+\end{enumerate}
+&
+\begin{enumerate}
+\item This issue has taken up a lot of time. This is partly due to the fact we only guessed 4 hours, this should have been 4 hours per person. Also, we took the time because we have made some pretty big changes to our future planning. We will deviate from the neural network we planned, because getting enough training data would be a master thesis size project, according to Cynthia. We had a meeting with Cynthia on Tuesday, to strategize our planning. We agreed on dropping the neural network, and search for a simpler way of getting past a cold start. Most of the training will be delegated to users, possibly in a gamification manner.
```

Comment:
"search for a simpler way of getting past a cold start."
could be changed to "develop a different mechanism to automatically update initial values."
This sounds better because  you don't get 'past' a cold start, you fix a cold start. Additionally the neural network was also developed to adjust song vectors over time, so this should be emphasized.